packages/components/src/DateTimePickers/DateTime/Manager/Manager.component.js

```diff
@@ -22,19 +22,23 @@ function ContextualManager(props) {
 	const initialState = extractParts(props.value, getDateOptions());
 	const [state, setState] = useState(initialState);
 
+	function onChange(event, payload) {
+		if (props.onChange) {
+			const { datetime, textInput, errors, errorMessage } = payload;
+			props.onChange(event, { datetime, textInput, errors, errorMessage });
+		}
+	}
+
 	useEffect(() => {
 		if (props.value !== state.datetime) {
 			const nextState = extractParts(props.value, getDateOptions());
 			setState(nextState);
+			if (nextState.errors && nextState.errors.length > 0) {
+				onChange(null, nextState);
+			}
```

Comment:
Can you add a comment above this code explaining the case please ?
To know that we want to propagate the error to the outside world when the new input is different (injected value) that is invalid.
I'm afraid that it's not that obvious and someone removes it in the future.

Reply:
added comment